services/skus/model/model.go

```diff
@@ -69,6 +79,65 @@ func (o *Order) IsStripePayable() bool {
 	return strings.Contains(strings.Join(o.AllowedPaymentMethods, ","), StripePaymentMethod)
 }
 
+var (
+	//TODO: populate
+	acceptedTokens []radom.AcceptedToken
+	acceptedChains []int64
+)
+
+// CreateRadomCheckoutSession - Create a Stripe Checkout Session for an Order
+func (o *Order) CreateRadomCheckoutSession(radomClient radom.Client, sellerAddress string) (CreateCheckoutSessionResponse, error) {
+
+	if len(o.Items) < 1 {
+		return CreateCheckoutSessionResponse{}, errors.New("failed to create checkout session, no order items")
+	}
+
+	successURI, ok := o.Items[0].Metadata["radom_success_uri"].(string)
+	if !ok {
+		return CreateCheckoutSessionResponse{}, errors.New("failed to create checkout session, no success url in sku")
+	}
+	cancelURI, ok := o.Items[0].Metadata["radom_cancel_uri"].(string)
+	if !ok {
+		return CreateCheckoutSessionResponse{}, errors.New("failed to create checkout session, no cancel url in sku")
+	}
+	productID, ok := o.Items[0].Metadata["radom_product_id"].(string)
+	if !ok {
+		return CreateCheckoutSessionResponse{}, errors.New("failed to create checkout session, no product id in sku")
+	}
+
+	// create a checkout session
+	resp, err := radomClient.CreateCheckoutSession(&radom.CheckoutSessionRequest{
+		SuccessURL:     successURI,
+		CancelURL:      cancelURI,
+		Currency:       "BAT",
+		AcceptedTokens: acceptedTokens,
+		AcceptedChains: acceptedChains,
+		SellerAddress:  sellerAddress,
+		Metadata: radom.Metadata(
+			[]radom.KeyValue{
+				{
+					Key: "brave-metadata",
+					Value: map[string]interface{}{
```

Comment:
This metadata value should be a string. I guess you could either serialize this map as a string if you're wanting to add more fields - or include them as separate objects in the array.